### PR TITLE
fix: Make the logic around `last_count` working

### PR DESCRIPTION
**Pull request by jhutar**
Did 2 changes here:
    
First is the important one where I removed `&& [[ $last_count -ge 0 ]]` as that was blocking changing last_count, because it was initialized to -1. Now it should work as expected.
    
Second I made sure warning is printed if we fail to obtain current count value.

### CODE REVIEW

File: ci-scripts/rhdh-setup/create_resource.sh [hunk 1]
```diff
@@ -353,8 +373,16 @@ get_token() {
   #shellcheck disable=SC2064
   trap "rm -rf $token_lockfile; exit" INT TERM EXIT HUP
 
+  echo "BEFORE"; cat "$token_field"
+
```
Comment (pmacik) [inline ci-scripts/rhdh-setup/create_resource.sh:377]: @jhutar I assume you have this here for debugging purposes... please remove it, unless you have a specific reason to keep it here.

File: ci-scripts/rhdh-setup/create_resource.sh [hunk 2]
```diff
@@ -374,6 +402,8 @@ get_token() {
     sleep 5s
   done
 
+  echo "AFTER"; cat "$token_field"
+
```
Comment (pmacik) [inline ci-scripts/rhdh-setup/create_resource.sh:406]: @jhutar I assume you have this here for debugging purposes... please remove it, unless you have a specific reason to keep it here.

**Review by pmacik — CHANGES_REQUESTED** [review]

File: ci-scripts/rhdh-setup/create_resource.sh (hunk 1 above)
Comment (jhutar) [reply ci-scripts/rhdh-setup/create_resource.sh]: Thank you Pavel. Please review now.

**Review by pmacik — APPROVED** [review]
/lgtm
/approve
Thx @jhutar

**Pull request merged**